ejb/src/main/java/org/wildfly/httpclient/ejb/RequestType.java

```diff
@@ -61,7 +61,7 @@ enum RequestType {
     /**
      * {@code OPEN} invocation type: used to create EJB session bean via HTTP protocol.
      */
-    OPEN(POST, EJB_OPEN_PATH),
+    CREATE_SESSION(POST, EJB_OPEN_PATH),
```

Comment:
What about the path name? EJB_CREATE_SESSION_PATH?

Reply:
This constant was eliminated.